web/07-continue.php

```diff
@@ -73,10 +73,15 @@
 }
 
 # start the workflow again
-chdir($folder);
-pclose(popen("$exec &", 'r'));
+if ($rabbitmq_host != "") {
+  $msg_exec = str_replace("\"", "'", $exec);
```

Comment:
This replaces all double quotes with single quotes in the command. So, `R_LIBS_USER="/path/to/R/library" "/usr/bin/R"` becomes `R_LIBS_USER='/path/to/R/library' '/usr/bin/R'`. This is because JSON doesn't support single-quoted strings, which would normally be the preferred way to escape this.

NOTE that bash _does not_ perform expansion inside single quotes, so any variables (i.e. `$var`) included in the command will _not_ be expanded.